lib/matplotlib/tests/test_patches.py

```diff
@@ -149,6 +149,40 @@ def test_rotate_rect_draw(fig_test, fig_ref):
     assert rect_test.get_angle() == angle
 
 
+@check_figures_equal(extensions=['png'])
+def test_dash_offset_patch_draw(fig_test, fig_ref):
+    ax_test = fig_test.add_subplot()
+    ax_ref = fig_ref.add_subplot()
+
+    loc = (0, 0)
+    width, height = (1, 1)
```

Comment:
```suggestion
    loc = (0.1, 0.1)
    width, height = (0.8, 0.8)
```
Added patches do not trigger autoscaling, so the recangle was exactly as large as the Axes and the dashes are almost invisible as they are overlaid by the spines.

Alternatively, one could fix the Axes limits, but it's simpler to start with a fitting rectangle in the first place.